### PR TITLE
Include post ids as relationships in tags

### PR DESCRIPTION
**Pull request by mansona**
Fixes https://github.com/ember-learn/empress-blog-ember-template/issues/27

To be able to display the "top 3 posts in the <new> tag" you need to have a reference from tag to the posts. This works fine if you don't have pagination turned on (as everything is stitched together in Ember Data) but you need the full list if you are using pagination and still want accurate relationships.

### CODE REVIEW

File: node-tests/tag-include-posts.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+describe('Tag Generator', function() {
+  beforeEach(async () => {
+    input = await createTempDir();
+  });
+
+  afterEach(async () => {
+    try {
+      await input.dispose();
+    } finally {
+      // do nothing
```
Comment (jaredgalanis) [inline node-tests/tag-include-posts.js:30]: Probs same as not including this try/finally? Did we need a catch for any reason?

File: index.js [hunk 2]
```diff
@@ -176,6 +191,7 @@ Please generate tags using 'ember generate tag your-tag-name'`);
         'image',
         'imageMeta',
       ],
+      references: [{ name: 'posts', type: 'contents' }],
```
Comment (jaredgalanis) [inline index.js:194]: Would it be valuable to include non custom-typed refs to demonstrate that string refs still work in combination?

Comment (mansona) [reply index.js:194]: in terms of it being valuable, we should for sure include examples in the docs for broccoli-static-site-json but this is the type of relationship that we need for this type in empress-blog ;) there are plenty of other examples in this file of non custom-typed refs if we are pointing people to empress-blog to see working examples 👍

File: node-tests/tag-include-posts.js (hunk 1 above)
Comment (mansona) [reply node-tests/tag-include-posts.js:30]: I'm including a bigger/better comment that explains why we want to do it this way 👍

**Comment by mansona** [comment]
Thanks for the review @jaredgalanis 🎉 time to get this merged 💪

**Pull request merged**